axiom/tests/test_php_packages.c

```diff
@@ -301,6 +302,74 @@ static void test_php_package_priority(void) {
   nr_php_packages_destroy(&hm);
 }
 
+static void nr_php_packages_itereate_callback(void* value,
+                                                const char* key,
+                                                size_t key_len,
+                                                void* user_data) {
+  nr_php_package_t* package = value;
```

Comment:
check if package is null?

Reply:
In this test it shouldn't ever be `NULL` so maybe failing would be a good thing?  What is primarily being tested here is that the callback is called with each package in the packages list.  I put a `NULL` guard for `buf` because I wanted to test that `NULL` is passed as `userdata` properly.